scripts/prepare_sector_network.py

```diff
@@ -2785,6 +2937,26 @@ def add_residential(n, costs, energy_totals):
     # Revise residential electricity demand
     buses = n.buses[n.buses.carrier == "AC"].index.intersection(n.loads_t.p_set.columns)
 
+    # Removing static loads from the time varying demand to preserve the distribution profile after normalization
+    static_load = (
+        n.loads.query('carrier.str.contains("electricity")').groupby("bus").p_set.sum()
+    )
+    n.loads_t.p_set.loc[:, buses] -= static_load

```

Comment:
Do we need this?

Reply:
To my understanding yes. The goal is avoiding distorting of the profile later but I didn't check it in details